Plugins/GitUIPluginInterfaces/GitRefsFlags.cs

```diff
@@ -1,18 +1,21 @@
-﻿namespace GitCommands.Git.Commands
+﻿namespace GitUIPluginInterfaces
 {
     /// <summary>
     /// Enums requestable in GitRefs() (multiple names can be appended)
     /// Compare to <see ref="GitRefType"/> for actual values of parsed GitRefs
     /// </summary>
-    public enum GetRefsEnum
+    public enum GitRefsFlags
     {
         None = 0,
 
-        Branches = 1 << 0,
+        // Local Branches
+        Heads = 1 << 0,
+
+        // Remote branches
         Remotes = 1 << 1,
         Tags = 1 << 2,
 
-        // All refs, including those abve but also (at least) stash, notes and bisect
+        // All refs, including those above but also (at least) stash, notes and bisect
         All = 1 << 3
```

Comment:
Should we add "stash, notes and bisect" as individual entries as well?

Reply:
Only add those we specifically need to get separately.

Any namespace can be added below refs/. 
Gerrit uses at least refs/for/ and refs/changes/ so all cannot be added.